src/CommunityToolkit.Maui/Options.cs

```diff
@@ -66,13 +66,25 @@ public void SetShouldEnableSnackbarOnWindows(bool value)
 			builder.ConfigureLifecycleEvents(events =>
 			{
 				events.AddWindows(windows => windows
+				.OnWindowCreated((_) =>
+				{
+					NumberOfWindows++;
```

Comment:
Are we going to go ahead and use `else if (Application.Current.Windows.Count is 1) ` and not switch to `Interlocked.Increment` and `Interlocked.Decrement`. I am happy either way but it would be nice to resolve this.